### PR TITLE
Add DomainRegistrantChange event types

### PR DESCRIPTION
**Pull request by wilson-dnsimple**
Add the struct type for the `domain.registrant_change` events.

Belongs to https://github.com/dnsimple/dnsimple-business/issues/1729

### CODE REVIEW

File: dnsimple/webhook/events.go
Comment (DXTimer) [inline dnsimple/webhook/events.go:1]: The event types also need to be added to the event types switch statement: https://github.com/dnsimple/dnsimple-go/pull/149/files#diff-22dab29957af6fba80649e7bc447dd4587d4d23963b73db5e7c2fc6803868025R10

**Review by DXTimer — CHANGES_REQUESTED** [review]

File: dnsimple/webhook/events.go
Comment (wilson-dnsimple) [reply dnsimple/webhook/events.go:1]: I think it already exists: https://github.com/dnsimple/dnsimple-go/blob/76d7b2cea116a1680ec2f95838aeb60249c0d330/dnsimple/webhook/events.go#L44

Comment (DXTimer) [reply dnsimple/webhook/events.go:1]: `domain.registrant_change:started` and `domain.registrant_change:cancelled` are missing.

**Review by DXTimer — CHANGES_REQUESTED** [review]
Can you also add tests for the new events in `events_test.go`?

**Review by DXTimer — APPROVED** [review]

**Pull request merged**